app/editor/src/components/wysiwyg/Wysiwyg.tsx

```diff
@@ -110,10 +114,32 @@ export const Wysiwyg = <T extends object>({
     }
   };
 
+  const tagMatch = /\[.*?\]/g;
+
+  const extractTags = (values: string[]) => {
+    return tags
+      .filter((tag) =>
+        values.some((value: string) => value.trim().toLowerCase() === tag.code.toLowerCase()),
+      )
+      .map((tag) => tag);
+  };
+
+  const updateTags = (html: string) => {
+    const stringValue = html.match(tagMatch)?.pop()?.toString().slice(1, -1);
+    const tagValues = stringValue?.includes(',')
+      ? stringValue?.split(',')
+      : stringValue?.split(' ') ?? [];
+    const tags = extractTags(tagValues);
+    if (!_.isEqual(tags, (values as any)?.tags)) setFieldValue('tags', tags);
```

Comment:
Are you intending to override the existing tags the user may have manually added?
If no tags are extracted this will result in clearing the tags.
We want to encourage users to use the Tags Component.  Your implementation would force them to instead use the square brackets and memorizing tag acronyms.

Reply:
Updated, and now it will be only adding but not removing, thanks